### ISSUE TITLE
Doesn't appear to work 0.6.7

### ISSUE THREAD

**Issue by kaihendry** [depcheck/depcheck#220]
```
~/eg$ depcheck --specials webpack,eslint            
Unused dependencies
* babel-core
* babel-loader
* babel-preset-es2015
* lodash
* moment
* moment-duration-format
* path
* request
* standard
* url
* uuid
* uuid-parse
* vue-loader
* vue-material
* vue-template-compiler
* vue-video
* vue-video-player
* webpack-dev-server
* zencoder
Unused devDependencies
* async-waterfall
* babel-eslint
* css-loader
* eslint
* eslint-config-standard
* eslint-config-standard-jsx
* eslint-plugin-html
* eslint-plugin-import
* eslint-plugin-node
* eslint-plugin-promise
* eslint-plugin-react
* eslint-plugin-standard
* file-loader
* json-loader
* style-loader
~/eg$ grep -r uuid src | grep require
src/components/Clip.vue:const uuidv4 = require('uuid/v4')
~/eg$ grep json-loader webpack.config.js
        loader: 'json-loader'
```

I use `uuid` as well as `json-loader`, so why the false positives?

**Comment by lijunle** [comment]
You can get more information with *--json* parameter.

2017年8月11日 下午12:29，"Kai Hendry" <notifications@github.com>写道：

**Comment by kaihendry** [comment]
```
{"dependencies":["babel-core","babel-loader","lodash","moment","moment-duration-format","path","request","standard","url","uuid","uuid-parse","vue-loader","vue-material","vue-template-compiler","vue-video","vue-video-player","zencoder"],"devDependencies":["async-waterfall","babel-eslint","css-loader","eslint-config-standard","eslint-config-standard-jsx","eslint-plugin-html","eslint-plugin-import","eslint-plugin-node","eslint-plugin-promise","eslint-plugin-react","eslint-plugin-standard","file-loader","json-loader","style-loader"],"missing":{},"using":{"babel-preset-es2015":["/home/hendry/spuul/video-ingestion/.babelrc"],"webpack":["/home/hendry/spuul/video-ingestion/package.json","/home/hendry/spuul/video-ingestion/webpack.config.js"],"webpack-dev-server":["/home/hendry/spuul/video-ingestion/package.json"],"eslint":["/home/hendry/spuul/video-ingestion/package.json"],"vue":["/home/hendry/spuul/video-ingestion/src/main.js","/home/hendry/spuul/video-ingestion/src/router/index.js"],"vue-async-computed":["/home/hendry/spuul/video-ingestion/src/main.js"],"vue-timeago":["/home/hendry/spuul/video-ingestion/src/main.js"],"vue-router":["/home/hendry/spuul/video-ingestion/src/router/index.js"],"aws-sdk":["/home/hendry/spuul/video-ingestion/src/services/aws.js"]},"invalidFiles":{},"invalidDirs":{}}
```

It's not giving me much to go on ...

**Comment by niftylettuce** [comment]
Doesn't work with node v8.x either

**Comment by niftylettuce** [comment]
Upon using `--json` I see there's some issue with `renderSync` in parseSASS?

<https://github.com/depcheck/depcheck/blob/master/src/parser/sass.js#L9>

```log
TypeError: Cannot read property 'renderSync' of null\n    at parseSASS (/Users/niftylettuce/.nvm/versions/node/v8.3.0/lib/node_modules/depcheck/dist/parser/sass.js:27:30)\n    at /Users/niftylettuce/.nvm/versions/node/v8.3.0/lib/node_modules/depcheck/dist/check.js:129:17\n    at tryToString (fs.js:513:3)
```

**Comment by mnkhouri** [comment]
@niftylettuce for your issue `Cannot read property 'renderSync' of null`, it is likely because you do not have `node-sass` installed, which is [required for parsing sass](https://github.com/depcheck/depcheck#syntax-support).

There have been a couple of versions (with several fixes) published since this issue was filed -- please don't hesitate to open a new issue if you have a problem!
